### ISSUE TITLE
TLS: tlsInitiator set pfx file，then client.SendAsync(),   a sspi error has occurred.

### ISSUE THREAD

**Issue by songhaideng** [fo-dicom/fo-dicom#1688]
### Expected behavior
tlsInitiator set pfx file，then client.SendAsync(),  expect successfully.

### Actual behavior
 a sspi error has occurred.  The received message is abnormal, or the format is incorrect.

### fo-dicom version and OS/platform
5.1.1.0

**Comment by songhaideng** [comment]
I set X509CertificateCollection to X509Certificate2Collection, then server can verify client Certificate file.

**Comment by amoerie** [comment]
So, this issue is resolved? 

@gofal Should we document somewhere that only X509Certificate2Collection is supported? Or should that work?

**Comment by gofal** [comment]
With certificates, the main work is not done in code, but before in communication between client and server, which type of certificate to use, do they need to have the same signing authority, what key-size is minimum required etc.
So I would not say, that it's a general documentation that X509Certificate2Collection has to be used. But there may be servers that only support certificates when used with advanced functions as provided in X509Certificate2Collection.
So when updating the documentation, this would be a more general documentation about certificates in .net and not about fo-dicom and how fo-dicom handles certificates.

**Comment by songhaideng** [comment]
yeah, I use DVTK-DICOM Validatation Tool as Server, Then I can send width Tls Successfully.

>
